docs/en/cookbook/working-with-datetime.rst

```diff
@@ -123,6 +123,11 @@ the UTC time at the time of the booking and the timezone the event happened in.
 
             return $converted;
         }
+        
+        private static function getUtc()
```

Comment:
How about adding a type hint here?

```suggestion
        private static function getUtc() : \DateTimeZone
```

Reply:
Force-commited proposed change. Also added dock-block for `self::$utc`.